### PR TITLE
Buildkite CI integration

### PR DESCRIPTION
**Pull request by igorgn**

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+steps:
+  - label: ":wrench: Build"
+    command:
+    - "nvm install"
```
Comment (asafkorem) [inline .buildkite/pipeline.yml:4]: why do we need `nvm install`?

Comment (igorgn) [reply .buildkite/pipeline.yml:4]: you can control which node version to use in '.nvmrc' file. Before each build that node will be installed

Comment (asafkorem) [reply .buildkite/pipeline.yml:4]: this project is pure iOS, so I guess we can remove this

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+14
```
Comment (asafkorem) [inline .nvmrc:1]: we can remove this file also
sorry for the nagging(!!)

**Review by asafkorem — APPROVED** [review]

**Comment by asafkorem** [comment]
Awesome!

**Pull request merged**